Symfony/CS/Fixer.php

```diff
@@ -158,9 +167,51 @@ public function fix(ConfigInterface $config, $dryRun = false, $diff = false)
         return $changed;
     }
 
-    public function fixFile(\SplFileInfo $file, array $fixers, $dryRun, $diff, FileCacheManager $fileCacheManager)
+    private function getNextFile(\Iterator $iterator, FileCacheManager $fileCacheManager)
```

Comment:
yes and no.
It returns the next file of iterator (starting with current) which is not a link and which is not cached.